.storybook/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+
+module.exports = {
+    env: {
+        browser: true,
+        es6: true,
```

Comment:
trailing comma not necessary?

Reply:
Trailing commas make [diffs look nicer](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8).